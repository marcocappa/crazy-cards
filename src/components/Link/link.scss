.crazy-card-link {
    background-color: #f4f4f4;
    border-radius: 0.5rem;
    border: 0.125rem solid #0f0a3a;
    color: #08051c;
    display: inline-block;
    font-family: "Source Sans Pro", Arial, sans-serif;
    font-size: 0.875rem;
    padding: 0.625rem 1.125rem;
    text-decoration: none;
    transition: border-color 0.2s ease-in-out;
}
