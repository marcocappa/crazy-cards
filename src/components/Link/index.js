import React from 'react';
import { Link } from 'react-router-dom';
import './link.scss';

const LinkComponent = (props) => (
  <Link {...props} className={'crazy-card-link'} />
);

export default LinkComponent;
