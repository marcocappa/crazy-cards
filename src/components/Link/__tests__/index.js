import React from 'react';
import { shallow } from 'enzyme';
import Link from '../index';

describe('<Link />', () => {
  it('should match snapshot and find classname', () => {
    const wrapper = shallow(<Link to="/" />);
    expect(wrapper).toMatchSnapshot();
    expect(wrapper.exists('.crazy-card-link')).toEqual(true);
  });
});
