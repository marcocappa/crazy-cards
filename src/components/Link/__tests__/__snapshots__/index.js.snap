// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Link /> should match snapshot and find classname 1`] = `
<Link
  className="crazy-card-link"
  to="/"
/>
`;
